Kernel/FileSystem/VirtualFileSystem.h

```diff
@@ -64,6 +70,9 @@ class VFS {
     VFS();
     ~VFS();
 
+    using AtFlags = int;
+    using OpenFlags = int;
```

Comment:
If we're going to have custom names for these, I think we should also make them strong types instead of just `int`.